checkout-ui-custom/src/_js/_customAddressForm.js

```diff
@@ -733,9 +733,6 @@ class fnsCustomAddressForm {
     _this.getCountryRule(country[1]).then(rules => {
       _this.addressrules = rules
       _this.updateFormFieldByCountry(_this.addressrules)
-      $("select[name='v-custom-state']").html(
-        `${_this.getRegions(country[0]).join('')}`
```

Comment:
Exact same behaviour as the first HTML instance; this line of code does not differ from

Reply:
these lines need to be there because the line 597 are not triggered again every country change

Reply:
Thank you @Arturo777 
Then it'll be a matter of checking data and selecting the desired value again. I'll amend the PR as soon as possible